components/Pagination/generateVisiblePages.js

```diff
@@ -0,0 +1,26 @@
+export function generateVisiblePages(currentPage, lastPage) {
```

Comment:
With this broken out, let's add some tests for this—especially as we can pass in a value and get a direct result from it. Aside from this, everything looks great.

Reply:
You could also do `export default function` because this is the only function and the file is named after it.